Tests/SonarScanner.MSBuild.Test/BootstrapperClassTests.cs

```diff
@@ -391,6 +402,7 @@ private IBootstrapperSettings MockBootstrapSettings(AnalysisPhase phase, bool de
 
             File.Create(Path.Combine(rootDir, "SonarScanner.MSBuild.Common.dll")).Close();
             File.Create(Path.Combine(rootDir, "SonarScanner.MSBuild.Tasks.dll")).Close();
+            File.Create(Path.Combine(rootDir, "Newtonsoft.Json.dll")).Close();
```

Comment:
this was hard to find - we basically have some test initialization inside
```
        [TestInitialize]
        public void MyTestInitialize()
```
and some other test initialization inside the verifier methods (!) : `CheckExecutionFails` and `CheckExecutionSucceeds`, which are calling this method `MockBootstrapSettings`